styles/basicfantasyrpg.css

```diff
@@ -374,7 +379,7 @@
 /* Styles limited to basicfantasyrpg sheets */
 
 .basicfantasyrpg.sheet.actor {
-  font-family: "Century Gothic", "TeX Gyre Adventor", "Soutane", var(--font-primary);
+  font-family: "Soutane", "Century Gothic", "TeX Gyre Adventor", var(--font-primary);
```

Comment:
On the sheet, Century Gothic is the preferred font. Soutane is the serif font we use for most of the UI, but the sheets use the sans-serif font.